### DISCUSSION TITLE
How to make the crawler wait 2.5 seconds before getting markdown?

### DISCUSSION THREAD

**Discussion by max-sym** [unclecode/crawl4ai#517] [Forums - Q&A]
I see that the `wait_for` parameter doesn't allow passing seconds to it, but only some condition / string. I simply want to make the crawler wait 2.5 seconds (to account for any new requests). I know about `networkidle0` (for JS playwright) but it doesn't fit my case. I need to literally wait 2.5 seconds. How can I do that? Thank you!

**Comment by unclecode** [comment]
Hi @max-sym, Thanks for using crawl4ai. To incorporate a delay before we fetch the HTML, which should be the part that I guess you need to work on, we can use the parameter `delay_before_return_html`, which is in seconds.
